src/qt/pivx/send.cpp

```diff
@@ -408,7 +413,7 @@ bool SendWidget::send(QList<SendCoinsRecipient> recipients)
     }
 
     dialog->deleteLater();
-    return false;
+    return true;
```

Comment:
won't this make so `updateEntryLabels` is called, even when the user doesn't confirm the send?

Reply:
yes, good catch